data_structures/queue/queue_with_stacks.py

```diff
@@ -23,4 +23,12 @@ def enqueue(self, val):
 
     def dequeue(self):
         """Remove a value from the front of the queue."""
-       pass
+        if self.front is not None:
+            first = self.front
+            self.front = self.front._next
+            self._size -= 1
```

Comment:
`front` is a Stack, so it does not have a `._next`. Also, you should only be using the `push()` and `pop()` methods for your Stack, not accessing the internals.